JetBrains prebuilds doc (#1945)
gitpod-io/website: gitpod/docs/ides-and-editors/goland.md

</figure>

**Important:** You must restart any started workspaces for your IDE preferences to take effect.

> ## Warming up in Prebuilds

Reviewer: Are we sure we want to adopt the "warming" terminology in Gitpod docs? I don't think we currently talk about warming elsewhere in the docs? 🤔  Maybe we should stick to prebuilds only here? WDYT?

```suggestion
## Indexing a JetBrains IDE using Prebuilds
```

Author: I can rename it to Indexing, but it is only a part of warming up, albeit most important. Space is talking about warming up as well: https://www.jetbrains.com/help/space/set-up-a-dev-evnvironment.html#step-2-warming-up-the-dev-environment

Author: warm-up:
>  a period of time when the IDE resolves project dependencies, builds indexes, and does other background activities

Author: I will rephrase introduction that it is not only about indexing.

> **Please note:** This feature is currently experimental and is a subject to change. [Send feedback](https://github.com/gitpod-io/gitpod/issues/6740).

> When you open the project {title} starts indexing to load modules and

Reviewer 2: ```suggestion
When you open the project, {title} starts indexing to load modules and
```

> enable the core funtionality like code completion and navigation.

Reviewer 2: ```suggestion
enable the core functionality like code completion and navigation.
```

Depending on the size of your project indexing speed can vary significantly.

Indexing can be speed up for you project by applying [general recommendations](https://www.jetbrains.com/help/go/indexing.html). Gitpod prebuilds allow to improve it further by indexing before you start a new workspace.

To leverage it:

- configure prebuilds for your repository, refer to [Prebuilds](/docs/prebuilds);
- enable {title} warming up in prebuilds by editing [.gitpod.yml](/docs/config-gitpod-file) in your repository:

```yaml
jetbrains:
  goland:

>     prebuilds:

Reviewer: Now I'm wondering whether "prebuilds" is the right term here ...

As this is enabling indexing, which is leveraging prebuilds? 

But this functionality is unrelated to things like the `init` command and core prebuild functionality? 🤔

Author: We can reconsider it later for sure. Indexing also is important part of warming up, but not a single, later we will also pre install plugins and so on.

For now I was aligning it with how we configure in other places like `github.prebuilds`: https://www.gitpod.io/docs/references/gitpod-yml#github

We can consider later to remove it from .gitpoy.ml and move to project settings instead on the dashboard.

      version: stable
```

- push changes to your Git repository to apply.

> `verison` property allows to control whether you would like to index only for `stable`, `latest` or `both` versions.

Reviewer 2: ```suggestion
`version` property allows to control whether you would like to index only for `stable`, `latest` or `both` versions.
```

> `verison` property allows to control whether you would like to index only for `stable`, `latest` or `both` versions.
> You can switch between `stable` and `latest` version of {title} in [your user preferences](https://gitpod.io/preferences).

Reviewer: ```suggestion
The `version` property allows you to control the indexing of JetBrains IDEs which are compatible with Gitpod. You can choose whether to index `stable`, `latest` or `both` IDE versions. Users can switch between `stable` and `latest` versions of {title} on the [user preferences](https://gitpod.io/preferences) page.
```
